Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -11,27 +11,47 @@
 
 using namespace amrex;
 
+std::unique_ptr< Vector<std::unique_ptr<amrex::MultiFab> > > m_costs;
+
 void
-WarpX::LoadBalance ()
+WarpX::LoadBalance (WarpX::LBType load_balance_type/*=DEFAULT*/)
```

Comment:
Verbose, defaults are defined in the header file.
```suggestion
WarpX::LoadBalance (WarpX::LBType load_balance_type)
```

Reply:
Removed.